.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - '6'
+  - '8'
```

Comment:
This is the test run with Node 6
https://travis-ci.org/rwjblue/sparkles-component/builds/551062566

Reply:
Ya, looks like ember-cli-fastboot-testing requires Node 8

Reply:
Seems fine to change, but can you make a separate PR for this part (and update engines in package.json to `8.* || 10.* || >= 12.*`)? I just want to make sure that gets labeled properly as a breaking change in the changelog...